src-v2/Not/Not.Blazor/TM/NotTimer.razor

```diff
@@ -0,0 +1,63 @@
+﻿@using System.Timers
+@implements IDisposable
+
+<div style="@($"color:{@_color};")">
+   @_displayTime
+</div>
+
+@code {
+    [Parameter]
+    public string StartTime { get; set; } = default!;
+    [Parameter]
+    public double StopTimerAtTime { get; set; } = default!;
+
+    private Timer _timer = default!;
+    private MudTheme _theme = new MudTheme();
+    private TimeSpan _time = default!;
+    private string _displayTime = default!;
+    private string _color = default!;
+
+    protected override void OnParametersSet()
+    {
+        _time = TimeSpan.Parse(StartTime);
+        FormatTime();
+        _timer = new Timer(1000);
+        _timer.Elapsed += OnTimerElapsed;
+        _timer.AutoReset = (_time > TimeSpan.FromMinutes(StopTimerAtTime));
+        _timer.Enabled = _time > TimeSpan.FromMinutes(StopTimerAtTime);
+    }
+
+    private void OnTimerElapsed(object? sender, System.Timers.ElapsedEventArgs e)
+    {
+        _time = _time.Subtract(TimeSpan.FromSeconds(1));
+        FormatTime();
+        if (_time <= TimeSpan.FromMinutes(StopTimerAtTime))
+        {
+            _timer.Enabled = false;
+        }
+        _displayTime = _time.ToString().Replace("-", " - ");
+        InvokeAsync(StateHasChanged);
```

Comment:
Inherit `NotComponent` and use `Render` insitead of manually `InvokeAsync(StateHasChanged)`